docs/en/setup/backend/log-otlp.md

```diff
@@ -1,4 +1,46 @@
 # OpenTelemetry Logging Format
 
-OpenTelemetry Collector supports to transport the logs to SkyWalking OAP.
-Read the doc on [Skywalking Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/skywalkingexporter) for a detailed guide.
+SkyWalking can receive logs exported from OpenTelemetry collector, the data flow is:
+
+```mermaid
+graph LR
+    B[OpenTelemetry SDK 1]
```

Comment:
I think this doesn't have to be OTEL SDK? OTEL collector has capabilities to transfer the log format accordingly, right?

Reply:
Polished the graph